compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/platform/PlatformContext.skiko.kt

```diff
@@ -115,7 +115,9 @@ interface PlatformContext {
     val parentFocusManager: FocusManager get() = EmptyFocusManager
     fun requestFocus(): Boolean = true
 
-    fun createDragAndDropManager(): PlatformDragAndDropManager = EmptyDragAndDropManager
+    fun createDragAndDropManager(
+        activeDragAndDropManager: () -> PlatformDragAndDropManager
+    ): PlatformDragAndDropManager = EmptyDragAndDropManager
```

Comment:
This API doesn't seem right. If you need to get `DragAndDropManager` in platform binding - add it as a property to `ComposeScene` interface